accounts/abi/bind/backends/blockchain.go

```diff
@@ -138,7 +138,16 @@ func (b *BlockchainContractBackend) callContract(call kaia.CallMsg, block *types
 	if call.AccessList != nil {
 		accessList = *call.AccessList
 	}
-	msg := types.NewMessage(call.From, call.To, 0, call.Value, call.Gas, call.GasPrice, call.Data,
+	gasPrice := common.Big0
+	if call.GasPrice != nil && (call.GasFeeCap != nil || call.GasTipCap != nil) {
+		return nil, errors.New("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified")
+	} else if call.GasPrice != nil {
+		gasPrice = call.GasPrice
+	} else if call.GasFeeCap != nil {
+		gasPrice = call.GasFeeCap
```

Comment:
Is it ok we directly use `call.GasFeeCap` while not considering gasTipCap for gasPrice?

Reply:
That sounds more accurate, but I'd like to just ignore that here because gasPrice is overwritten to 0 anyway.